src/common/ConversationSummary.js

```diff
@@ -1,71 +1,8 @@
 import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
-import moment from 'moment';
+import { formatDate, formatName } from '../lib/Formatters';
 import Colors from '../constants/Colors';
 
-const NAME_SOURCE = {
-  NAME: 'NAME',
-  EMAIL: 'EMAIL',
-  PHONE: 'PHONE',
-};
-
-const formatDate = ({ date }) => {
-  if (!date) {
-      return '';
-  }
-
-  const selectedDate = moment(date);
-
-  if (selectedDate.isSame(moment(), 'd')) { // Is today
-      return `${selectedDate.format('[Today] h:mm a')}`;
-  } else if (selectedDate.isAfter(moment().subtract(1, 'd').startOf(), 'd')) { // Is yesterday
-      return `${selectedDate.format('[Yesterday] h:mm a')}`;
-  } else if (selectedDate.isAfter(moment().startOf('year'), 'd')) { // Is this year
-      return `${selectedDate.format('MMM D h:mm a')}`;
-  } else { // Previous years
-      return `${selectedDate.format('MMM D YYYY h:mm a')}`;
-  }
-}
-
-const formatPhoneNumber = ({ phoneNumber }) => {
```

Comment:
No longer needed here since it is not being imported? or it was not doing anything?

Reply:
I still needed to copy it over to the formatters lib, since it's used in formatName()